docs/docs/en/guide/howto/general-setting.md

```diff
@@ -0,0 +1,22 @@
+# General Setting
+
+## Language
+
+DolphinScheduler supports two types of built-in language which include `English` and `Chinese`. You could click the button
+on the top control bar named `English` and `Chinese` and change it to another one when you want to switch the language. All
+your pages of DolphinScheduler's language will be changed when you switch the language from one to another.
+
+## Theme
+
+DolphinScheduler supports two types of built-in theme which include `Dark` and `Light`. When you want to change the theme
+of DolphinScheduler, all you have to do is click the button named `Dark`(or `Light`) on the top control bar and on the left
+of to [language](#language) control button.
+
+## Time Zone
+
+DolphinScheduler support time zone setting. The build-in time zone is based on the server you run DolphinScheduler. You could
+click the button on the right of the [language](#language) button and then click `Choose timeZone` to choose the time zone
+you want to switch. All time related components will change their time zone after you change the time zone.
```

Comment:
All time related components will change their time zone after you change the time zone.
->
All time related components will adjust their time zone according to the time zone setting you select.